.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["env"]
+  "presets": ["@babel/env"]
```

Comment:
Should we also add `"sourceType": script` here similar to https://github.com/metarhia/jstp/pull/378?

Reply:
@belochub I don't think we should, since this option is undocumented and it won't change anything: we don't use webpack there, so we don't really care about CJS vs ESM in this case.